### ISSUE TITLE
Tile Inspector highlighted object changes when changing order

### ISSUE THREAD

**Issue by danielmeinert** [OpenRCT2/OpenRCT2#20722]
### Operating System

Windows 10, 64 bit

### OpenRCT2 build

OpenRCT2, v0.4.5-257-g9d6a54b (9d6a54b on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

When changing the position of an object on the tile order the highlighting of gets lost and changes to another object on the tile.

![Animation](https://github.com/OpenRCT2/OpenRCT2/assets/39702411/56aea921-7a90-4687-8648-c43122954beb)

### Steps to reproduce

Build more than one object on tile.
Select object in tile inspector.
Change ordering.
Observe that highlight changed to other object.

**Comment by KawkMob** [comment]
The issue does not seem to be present in the current build. Possibly fixed by https://github.com/OpenRCT2/OpenRCT2/issues/20481

**Comment by ZehMatt** [comment]
> The issue does not seem to be present in the current build. Possibly fixed by #20481

Yeah most likely, let us know if there is still a problem, closing this.
